integration_tests/snaps/meson-hello/snapcraft.yaml

```diff
@@ -0,0 +1,18 @@
+name: meson-hello
+version: 0.1
+summary: test the meson plugin
+description: |
+  This is a basic meson snap. It just prints a hello world.
+  If you want to add other functionalities to this snap, please don't.
+  Make a new one.
+icon: icon.png
```

Comment:
Please remove this entry and the .png file.
This optional, and it's good to keep the integration tests as small as possible.